script/Arcueid/Day 8/08_00_ARC08_3.txt

```diff
@@ -256,7 +256,8 @@ The other students haven't noticed her,
 -- “正真正銘、根っからの<破壊魔|デストロイヤー>め……！”
 // Not voiced
 // Destroyer reads kinda whack but can't think of anything better rn and it is ruby en
-I swear to god, she is a destroyer, through and through!
+// Agent of chaos?
+I swear to god, she is an agent of chaos, through and through!
```

Comment:
Definitely improved. Through and through feels _slightly_ off as an emphasizer here? Also I think singular God needs to be capitalized. Playing around with some alternates - 
> I swear to God, this woman is chaos incarnate!
> I swear to God, this woman is a walking natural disaster.
> This damned woman is a veritable walking disaster!
> That woman is a goddamn walking disaster!

Reply:
Any of these work for me, honestly.
Uhhh, let's just go with No. 3